README.md

```diff
@@ -340,15 +336,19 @@ npm run add-provider-module
 
 ### Step 2: Submit your Task to Gelato via your GelatoUserProxy
 
-The script we will run submits our `Task` with a limit of three executions to Gelato, to limit the number of automatic trades we want to occur. Spelled out this will mean:
+In this example, the user wants to instruct Gelato to execute 3 trades in total on its behalf, each swapping 1 DAI to KNC after a certain time has passed.
```

Comment:
You switched person from "we want to instruct" to "the user wants to" - let's keep it "we want to" like everywhere else (more engaging)

Reply:
I wanted to make the distinction between "we are now the user" vs "we were the provider". I'll switch it back to we with (the user) in brackets to make it clear